_docs/setup/ad-connect.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Active Directory Integration
+category: setup
+show_in_sidebar: true
+---
+
+## Active Directory Integration
+
+OmniOS can integrate itself into an active directory domain as an smb file server and it can use an active directory server for both login authentication as well as to provide name services for user accounts.
+
+### Preparing the System
+
+Use AD as your dns server
+
+```terminal
+# cat <<CFG_END >/etc/resolv.conf
+nameserver 10.1.1.1
+domain my-ad-domain.local
+search my-ad-domain.local
+CFG_END
+```
+
+Enable ntp as precise time is essential for kerberos operation.
+
+```terminal
+# svcadm enable network/ntp
+```
+
+### Hooking up Kerberos
+
+Install Kerberos support
+
+```terminal
+# pkg install system/security/kerberos-5
+```
+
+Setup Kerberos for MS Active Directory (ms_ad)
+
+```terminal
+# kclient -T ms_ad          
+# kinit -V administrator         
+```
+
+### Enabeling SMB Filesharing
+
+To make sure we have the same user id numbers everywhere we want to keep them in Active Direcory. 
+
+```terminal
+# svccfg -s svc:/system/idmap setprop \ config/directory_based_mapping=astring: idmu
+# svcadm refresh svc:/system/idmap
+```
+
+Make sure you have the following properties configured for every user in your active directory:
+
+    * uid (username)
+    * uidNumber 
+    * gitNumber
+    * homeDirectory
+    * loginShell
+    * gecos
+
+![Active Directory Configuration](./ad-cfg.png)
```

Comment:
I don't suppose it matters, but up to now images have been under /assets/images/